core/src/test/java/org/apache/druid/common/guava/ComplexSequenceTest.java

```diff
@@ -88,20 +88,15 @@ public Integer accumulate(Integer accumulated, Integer in)
     return Sequences.concat(Arrays.asList(sequences));
   }
 
-  private final BinaryFn<Integer, Integer, Integer> plus = new BinaryFn<Integer, Integer, Integer>()
-  {
-    @Override
-    public Integer apply(Integer arg1, Integer arg2)
-    {
-      if (arg1 == null) {
-        return arg2;
-      }
-
-      if (arg2 == null) {
-        return arg1;
-      }
+  private final CombiningFunction<Integer> plus = (arg1, arg2) -> {
```

Comment:
Please make this function static. I would call it PLUS_NULLABLE to highlight why it's not just `Integer::sum`.